server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLReader.java

```diff
@@ -343,6 +343,21 @@ private void parseContainer(XMLExtendedStreamReader reader, PathAddress subsyste
                 }
             }
         }
+
+        addSyntheticHealtheOperation(operations, containerAddress);
```

Comment:
hahah it's a typo :) Anyway I just removed this method... Could you please hold on with the review for ~30 mins? I'll upload new version (once I fix it :D)